libbpf-tools/biolatency.bpf.c

```diff
@@ -52,15 +57,27 @@ int trace_rq_start(struct request *rq)
 }
 
 SEC("tp_btf/block_rq_insert")
-int BPF_PROG(block_rq_insert, struct request_queue *q, struct request *rq)
+int block_rq_insert (u64 *ctx)
 {
+	/**
+	 * commit a54895fa (v5.11-rc1) changed tracepoint argument list
+	 * from TP_PROTO(struct request_queue *q, struct request *rq)
+	 * to TP_PROTO(struct request *rq)
+	 */
+	struct request *rq = LINUX_KERNEL_VERSION >= KERNEL_VERSION(5, 11, 1) ? (void *)ctx[0] : (void *)ctx[1];
```

Comment:
it's a very long line, can you wrap it? Or maybe cleaner is to do it with if/else after variable declaration.

plus also empty line between variable declarations and the rest of function's code